addon/components/base-chart-component.js

```diff
@@ -103,6 +103,10 @@ export default Component.extend({
     didReceiveAttrs() {
         this._super(...arguments);
 
+        if (this.get('isDestroyed') || this.get('isDestroying')) {
+            return;
+        }
```

Comment:
Was there an issue that this code solves? I wouldn't think that anything outside of the runloop would be calling `didRecieveAttrs`

Reply:
oh sorry missed your description... still seems odd that didRecieveAttrs would be triggered for a destroyed component